From 3d0a9a24bda2d884ad5256cd5deef210470f7a1c Mon Sep 17 00:00:00 2001
From: "Pottimurthy, Sathya Narayana" <sathya.narayana.pottimurthy@intel.com>
Date: Tue, 20 Apr 2021 19:13:07 -0400
Subject: [PATCH 2/7] mcdma: DPDK devbind script patches to bind MCDMA device

DPDK devbind script patches to bind MCDMA device

Signed-off-by: Pottimurthy, Sathya Narayana
<sathya.narayana.pottimurthy@intel.com>
Title: mcdma: DPDK devbind script patches to bind MCDMA device
---
 usertools/dpdk-devbind.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/usertools/dpdk-devbind.py b/usertools/dpdk-devbind.py
index b1d1498..2b53232 100755
--- a/usertools/dpdk-devbind.py
+++ b/usertools/dpdk-devbind.py
@@ -45,10 +45,14 @@
               'SVendor': None, 'SDevice': None}
 intel_ioat_skx = {'Class': '08', 'Vendor': '8086', 'Device': '2021',
               'SVendor': None, 'SDevice': None}
+intel_ioat_icx = {'Class': '08', 'Vendor': '8086', 'Device': '0b00',
+              'SVendor': None, 'SDevice': None}
 intel_ntb_skx = {'Class': '06', 'Vendor': '8086', 'Device': '201c',
               'SVendor': None, 'SDevice': None}
+intel_mcdma = {'Class': '06', 'Vendor': '8086', 'Device': '1172',
+              'SVendor': None, 'SDevice': None}
 
-network_devices = [network_class, cavium_pkx, avp_vnic, ifpga_class]
+network_devices = [network_class, cavium_pkx, avp_vnic, ifpga_class, intel_mcdma]
 baseband_devices = [acceleration_class]
 crypto_devices = [encryption_class, intel_processor_class]
 eventdev_devices = [cavium_sso, cavium_tim, octeontx2_sso]
@@ -318,6 +322,8 @@ def device_type_match(dev, devices_type):
         param_count = len(
             [x for x in devices_type[i].values() if x is not None])
         match_count = 0
+        if dev["Vendor"] == "1172" :
+	    return True
         if dev["Class"][0:2] == devices_type[i]["Class"]:
             match_count = match_count + 1
             for key in devices_type[i].keys():
-- 
1.8.3.1

